src/bikeracks/__init__.py

```diff
@@ -78,7 +95,6 @@ def submitFeedback():
                 # writing the data rows 
                 csvwriter.writerow(row)
         except Exception as e:
-            print(e)
             raise e
```

Comment:
This try/except does nothing but reraise the exception? Scrap it.

Reply:
Done.